compiler/src/dotty/tools/dotc/interactive/Completion.scala

```diff
@@ -92,7 +92,14 @@ object Completion {
         }.getOrElse("")
 
       case (ref: untpd.RefTree) :: _ =>
-        if (ref.name == nme.ERROR) ""
+        if (ref.name == nme.ERROR) {
+          val content = ref.source.content()
+          // if the error resulted from unclosed back tick
+          if content(ref.span.start) == '`' then
+            content.slice(ref.span.start + 1, ref.span.end).mkString
```

Comment:
I'm just wondering whether this is enough to tell that an unclosed backtick was the actual problem. Could we have a different kind of error if the backtick was closed?

Reply:
It should be enough especially if backtick is first, otherwise anything inside backticks should be ok.